src/ui/components/usersListTable/UsersListTable.scss

```diff
@@ -57,44 +70,50 @@ $container-padding-v: 24px;
 
     &.passwordless {
       color: var(--color-danger);
-      background-color: var(--color-danger-opacity-20);
+      background-color: var(--color-danger-bg);
     }
     &.emailpassword {
-      color: var(--color-success);
-      background-color: var(--color-success-opacity-20);
+      color: var(--color-success-1);
+      background-color: var(--color-success-1-bg);
     }
     &.thirdparty {
-      color: var(--color-light);
-      background-color: var(--color-light-opacity-20);
-      &.google, &.apple {
+      color: var(--color-success-0);
+      background-color: var(--color-success-0-bg);
+      &.google,
+      &.apple {
         color: var(--color-danger-1);
-        background-color: var(--color-danger-1-opacity-20);
+        background-color: var(--color-danger-1-bg);
         span {
           text-transform: capitalize;
         }
       }
-      &.github, &.facebook {
-        color: var(--color-success-1);
-        background-color: var(--color-success-1-opacity-20);
+      &.github,
```

Comment:
Looks like styling for Github is broken

<img width="160" alt="Screenshot 2022-08-11 at 11 50 53 AM" src="https://user-images.githubusercontent.com/18233774/184075284-ce9f6333-901e-44e3-a64a-8e483e50ce7e.png">

The same issue happens with unknown providers as well

<img width="175" alt="Screenshot 2022-08-11 at 11 48 41 AM" src="https://user-images.githubusercontent.com/18233774/184075327-2303bf0a-3a8d-46fb-aae3-ae579b07616a.png">

Reply:
I think you have solved this one already